lighthouse-core/gather/drivers/extension.js

```diff
@@ -142,18 +141,16 @@ class ExtensionDriver extends Driver {
           return reject(chrome.runtime.lastError);
         }
 
-        this.url = tabs[0].url;
-        resolve(tabs[0].id);
+        resolve(tabs[0]);
```

Comment:
while you're in here, should this be caching the tab found and always resolving on it in subsequent calls? Seems bad that a call to `getCurrentTabURL()` could potentially return a different tab than when `connect()` is finally called